components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -571,6 +571,33 @@ def testGetImage(self, gatewaywrapper, author_testimg_tiny):
         assert image.getOwnerOmeName == testImage.getOwnerOmeName
         assert image.getThumbVersion() is not None
 
+    @pytest.mark.parametrize("load_pixels", [True, False])
+    @pytest.mark.parametrize("load_channels", [True, False])
+    def testGetImageLoadPixels(self, load_pixels, load_channels,
+                               gatewaywrapper, author_testimg_tiny):
```

Comment:
@jburel Re: "what is author?". 
author_testimg_tiny is a fixture created by import of 'tinyTest.d3d.dv'